spec/acceptance/ini_setting_spec.rb

```diff
@@ -72,13 +69,13 @@
         apply_manifest(pp, :catch_changes => true)
       end
 
-      it_behaves_like 'has_content', "#{tmpdir}/ini_setting.ini", pp, "four = five\n[one]\ntwo = three"
+      it_behaves_like 'has_content', "#{tmpdir}/ini_setting.ini", pp, /four = five\n\n\[one\]\ntwo = three/
```

Comment:
Every time I ran it with a regex it was reporting two returns.  Not sure if the grep was reporting differently or stdout, but it was consistent with all platforms on that failure.  Tested on CentOS 6, Win 2008/2012 and Solaris 10.